pkg/api/types.go

```diff
@@ -252,6 +255,9 @@ type PersistentVolumeSpec struct {
 	AccessModes []AccessModeType `json:"accessModes,omitempty"`
 	// holds the binding reference to a PersistentVolumeClaim
 	ClaimRef *ObjectReference `json:"claimRef,omitempty"`
+	// SecretRef is a reference to a Secret in the system.
+	// used for persistent volume authentication
+	SecretRef *ObjectReference `json:"secretRef,omitempty"`
```

Comment:
It would have to be when the PV is created.  

The admin would create a Secret in a namespace they control.  Then they create the PV and use that secretRef.